translator/trace/jaeger/protospan_to_jaegerthrift.go

```diff
@@ -444,6 +446,10 @@ func ocSpanAttributesToJaegerTags(ocAttribs *tracepb.Span_Attributes) []*jaeger.
 		}
 
 		jTag := &jaeger.Tag{Key: key}
+		if key == errorTag {
+			jTag, jTags = handleError(jTag, jTags)
```

Comment:
Unclear what this code is doing.

Reply:
I was confused as well. It is a bit cryptic. If we do accept this feature, this should be re-worked as I suggested above.

Reply:
i will add inline code to make it more readable and add some comments.

Reply:
Refactoring code [WorkInProgress]